test/header_parser_tests/expected_bindings/_expected_objc_basic_types_bindings.dart

```diff
@@ -0,0 +1,210 @@
+// AUTO GENERATED FILE, DO NOT EDIT.
+//
+// Generated by `package:ffigen`.
+import 'dart:ffi' as ffi;
+import 'package:ffi/ffi.dart' as pkg_ffi;
+
+/// Opaque Dependencies Test
+class NativeLibrary {
+  /// Holds the symbol lookup function.
+  final ffi.Pointer<T> Function<T extends ffi.NativeType>(String symbolName)
+      _lookup;
+
+  /// The symbols are looked up in [dynamicLibrary].
+  NativeLibrary(ffi.DynamicLibrary dynamicLibrary)
+      : _lookup = dynamicLibrary.lookup;
+
+  /// The symbols are looked up with [lookup].
+  NativeLibrary.fromLookup(
+      ffi.Pointer<T> Function<T extends ffi.NativeType>(String symbolName)
+          lookup)
+      : _lookup = lookup;
+
+  int __darwin_check_fd_set_overflow(
```

Comment:
Where do these functions come from? I see only the struct in the in the header file.

Reply:
parser.dart:70. I have to auto-include some system headers to get it to parse. Some of the internal built in types are defined in there.

Reply:
I guess we should be able to exclude these declarations by matching on its source file, see this -
https://github.com/dart-lang/ffigen/blob/dd3e08945a5be158981c3e1e852ef4b6e56c5547/lib/src/header_parser/includer.dart#L72

Reply:
I think Ideally we should, but I guess we cannot if these might be needed by users.

Reply:
I think we should not generate them until we have a use case in a follow up CL. Otherwise it is not clear what is the mapping between the header file + config yaml file to the output.

Reply:
Done